functions/report_push/jira_events.py

```diff
@@ -37,7 +39,8 @@ def get_all_issues(jira_project_key):
     return issues
 
 
-def create_new_bug(description, replaced_allure_links, summary, jira_project_key):
+def create_new_bug(description, replaced_allure_links, summary,
+                   jira_project_key):
     print(f'Will be created bug with name[{summary}]')
```

Comment:
Let's create new task to replace prints to log

Reply:
we already have this task